scripts/test.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 if [[ $WIRESPEC_BUILD_ALL = true || $WIRESPEC_BUILD_MAC = true ]]; then
   echo "Test macOS artifact"
-  ./compiler/$artifactName/build/bin/$macosArch/releaseExecutable/$artifactName.kexe "$(pwd)"/types $languages
+  ./compiler/$artifactName/build/bin/$macosArch/releaseExecutable/$artifactName.kexe $languages "$(pwd)"/types
```

Comment:
why do you need to give the languages first? Maybe also provide an -i for input?